base/dict/src/main/resources/org/openscience/cdk/dict/data/descriptor-algorithms.owl

```diff
@@ -377,6 +377,18 @@
         <isClassifiedAs rdf:resource="#molecularDescriptor"/>
     </Descriptor>
 
+    <Descriptor rdf:ID="nSpiroAtom">
+        <rdfs:label>SpiroAtoms Count</rdfs:label>
+        <dc:contributor rdf:resource="#mf"/>
```

Comment:
please update to state your IRI... (which is #rguha)

Reply:
Good point. Updated